package.json

```diff
@@ -44,7 +46,7 @@
     "start": "concurrently \"npm run watch-css\" \"electron .\"",
     "style": "eslint -c .eslintrc *.js scripts/* src/*.js src/**/*.jsx test/**/*.js",
     "unit-tests": "xvfb-maybe electron-mocha --renderer --recursive --compilers js:babel-register test",
-    "test": "npm run-script style && NODE_ENV=test npm run-script unit-tests",
+    "test": "npm run-script style --silent && NODE_ENV=test npm run-script unit-tests --silent",
```

Comment:
It gets rid of this annoying npm error

![img](https://probablycorey-monosnap.s3.amazonaws.com/probablycoreyCoreys-MacBook-Pro_githubgh-notifications-snoozer_2016-11-16_16-52-09.png)